src/vaishnavi_SeleniumBasics/EmployeeDetails.java

```diff
@@ -0,0 +1,57 @@
+package vaishnavi_SeleniumBasics;
+
+public class EmployeeDetails {
+
+	private int empId;
+	private String empName;
+	private int empManagerId;
+	private String empDepartmentName;
+
+	public void setEmpId(int empId) {
+		this.empId = empId;
+	}
+
+	public int getEmpId() {
+		return empId;
+	}
+
+	public void setEmpName(String empName) {
+		this.empName = empName;
+	}
+
+	public String getEmpName() {
+		return empName;
+	}
+
+	public void setEmpManagerId(int empManagerId) {
+		this.empManagerId = empManagerId;
+	}
+
+	public int getEmpManagerId() {
+		return empManagerId;
+	}
+
+	public void setEmpDepartmentName(String empDepartmentName) {
+		this.empDepartmentName = empDepartmentName;
+	}
+
+	public String getEmpDepartmentName() {
+		return empDepartmentName;
+	}
+	
+	@Override
+	public int hashCode() {
+		return empId;
+	}	
+
+	@Override
+	public boolean equals(Object obj) {
+		EmployeeDetails employeeDetails1 = (EmployeeDetails) obj;
+		
```

Comment:
hi @VaishnaviVJun20 
i feel you should add below condition as well
if (this == obj)
       return true;